src/main/java/overlap_crap_fix_locations_later/application_business/CalculateSectionHoursInteractor.java

```diff
@@ -0,0 +1,26 @@
+package overlap_crap_fix_locations_later.application_business;
```

Comment:
If this is your code, could you please rename the package to something more descriptive?

Reply:
Done! My bad, forgot to fix this earlier. But this is why we do prs!